src/geometry/manifold/algebra/smooth_functions.lean

```diff
@@ -80,6 +89,17 @@ instance smooth_map_group {G : Type*} [group G] [topological_space G]
   mul_left_inv := λ a, by ext; exact mul_left_inv _,
   .. smooth_map_monoid }
 
+@[simp, to_additive]
+lemma smooth_map.coe_inv {G : Type*} [group G] [topological_space G]
+  [charted_space H' G] [lie_group I' G] (f : C^∞⟮I, N; I', G⟯) :
+  ⇑f⁻¹ = f⁻¹ := rfl
+
+@[simp, to_additive]
+lemma smooth_map.coe_div {G : Type*} [group G] [topological_space G]
+  [charted_space H' G] [lie_group I' G] (f g : C^∞⟮I, N; I', G⟯) :
+  ⇑(f / g) = f / g :=
+by simp only [div_eq_mul_inv, smooth_map.coe_inv, smooth_map.coe_mul]
```

Comment:
Out of scope for this PR, but it might be a good idea to fix `smooth_map_group` above to make this defeq, by adding `inv := ...`

Reply:
I agree this should be done but somehow the obvious things didn't immediately work for me.

I'm out of time for today so I'll go ahead and merge but I'll take a look again tomorrow.